### ISSUE TITLE
the SR image of SRGAN BSD100

### ISSUE THREAD

**Issue by qingchuanhuajuan** [xinntao/ESRGAN#12]
Thank you for your wonderful work. I am also very interested in GAN, how do you get the SR image of SRGAN's BSD100?

**Comment by xinntao** [comment]
Did you mean get the original SRGAN's results on BSD100? 
You can download from https://twitter.app.box.com/s/lcue6vlrd01ljkdtdkhmfvk7vtjhetog

**Comment by qingchuanhuajuan** [comment]
Yes, I want the original SRGAN super resolution image in the BSD100 test set (X4). But I can't open this url. If it's convenient, you can send it to my mailbox. Thank you very much!（chjiaojiaoqiao@gmail.com）
